### PR TITLE
fix starpuff

### PR DESCRIPTION
**Pull request by Suyinkee**

### CODE REVIEW

File: src/components/templates/Home/index.js [hunk 1]
```diff
@@ -15,8 +15,14 @@ import turtle from 'assets/Images/turtle.png';
 import starFish from 'assets/Images/starfish.svg';
 import FaAngleRight from 'react-icons/lib/fa/angle-right';
 
+global.starpuff_count = 0;
```
Comment (andiwinata) [inline src/components/templates/Home/index.js]: i'm not sure if we should introduce `global` variable, we can always extract it from firebase right?

Comment (andiwinata) [reply src/components/templates/Home/index.js]: also use `camelCase` instead using `underscore_variable`

**Review by andiwinata — COMMENTED** [review]
did you do this by youself @Suyinkee ? if yes, that's impressive you know where to put stuff

File: src/components/templates/Home/index.js (hunk 1 above)
Comment (ghost) [reply src/components/templates/Home/index.js]: @andiwinata I think this was @chengcai616's commit that @Suyinkee was trying to get reviewed?

**Pull request merged**